modules/nw-operator-cr.adoc

```diff
@@ -315,8 +320,21 @@ One of the following additional audit log targets:
 |====
 // end::policy-audit[]
 
+.`gatewayConfig` object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`routingViaHost`
+|`boolean`
+|Set this field to `true` to send egress traffic from pods to the host networking stack.
+For highly-specialized installations and applications that rely on manually configured routes in the kernel routing table, you might want to route egress traffic to the host networking stack.
```

Comment:
Not sure if we want to mention the hardware offload part, but it that feature is not documented at all, then maybe it doesn't make sense to be included.

Reply:
@tssurya , not urgent, but I just added the interaction with the OVS HW offload.  (I thought I already did that and I thought I already asked you to review.)

I synced with Chris T on the customer-facing name for the HW offload feature.  Please let me know if I've goofed the interaction.

Reply:
ack thanks Mike! its looking good to me